src/borg/constants.py

```diff
@@ -66,6 +66,12 @@
 EXIT_WARNING = 1  # reached normal end of operation, but there were issues
 EXIT_ERROR = 2  # terminated abruptly, did not reach end of operation
 
+# never use datetime.isoformat(), it is evil. always use one of these:
```

Comment:
Is there a linter that can check stuff like this automatically?

Reply:
i don't know.